keen/explorer
# Disabled buttons and message for actions not supported yet by certain analysis types

## What does this PR do? How does it affect users?
It shows the query actions buttons regardless of whether the analysis type is supported or not and just disables them and shows a simple message to let users know these types are not yet savable. Lots of people have bene asking why "Funnels are not supported for saving in Explorer."

## How should this be tested?

Try setting the analysis types to `funnel`, `extraction` (and setting the query builder to "bulk email extraction"), and then other analysis types and notice the change in the query actions section of the explorer.

Step through the code line by line. Things to keep in mind as you review: 
 - Are there any edge cases not covered by this code? 
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:
